backend/src/models/reports/Tab_5_rpt_CA_ConsultingServiceUtilization-FY-Summary.js

```diff
@@ -0,0 +1,159 @@
+// import knex database connection and utilities
+const { knex } = require("@database/databaseConnection")();
+const log = require("../../facilities/logging")(module.filename);
+const _ = require("lodash");
+
+/**
+ *
+ * Retrieves invoiced amount towards each type of contract resource
+ * (Application Design-Development, Application Management, Architecture,
+ * Business Analysis and Process Consultation etc.)
+ * Based on the Competencies in the SRI
+ *
+ * @param   {number}               fiscal - The fiscal year to grab data for
+ * @returns {Promise<{report: *}>}
+ */
+const queries = {
+  // returns the fiscal year.
+  fiscal_year: (fiscal) =>
+    knex("fiscal_year").select("fiscal_year").where("fiscal_year.id", fiscal).first(),
+
+  // returns the report data.
+  report: (fiscal) =>
+    knex
+      .with(
+        "fixed_price_resource_count",
+        knex.raw(
+          `SELECT contract_id,
+          count(*) fixed_price_resource_count
+        FROM contract_resource
+        WHERE hours IS NULL
+        GROUP BY contract_id`
+        )
+      )
+      .with(
+        "fixed_price_invoice_total",
+        knex.raw(
+          `SELECT i.contract_id,
+          sum(id.unit_amount * id.rate) fixed_price_amount
+        FROM invoice i
+          INNER JOIN invoice_detail id ON i.id = id.invoice_id
+          INNER JOIN contract_deliverable cd ON id.contract_deliverable_id = cd.id
+        WHERE cd.is_expense = FALSE
+        GROUP BY i.contract_id`
+        )
```

Comment:
```suggestion
        knex.raw(`
          SELECT
            i.contract_id,
            sum(id.unit_amount * id.rate) fixed_price_amount
          FROM invoice i
            INNER JOIN invoice_detail id ON i.id = id.invoice_id
            INNER JOIN contract_deliverable cd ON id.contract_deliverable_id = cd.id
          WHERE cd.is_expense = FALSE
          GROUP BY i.contract_id
        `)
```
Another nitpick but the indentation looks a little off on some of these queries. This is how I would find them easiest to read.